CHANGELOG.md

```diff
@@ -49,6 +49,16 @@ Current
 
 ### Changed:
 
+- [`RequestLog::switchTiming` has been deprecated]((https://github.com/yahoo/fili/pull/141)
```

Comment:
Not sure where the other comment I left went, but this note should move into the `deprecated` section